rapids_build_backend/config.py

```diff
@@ -1,21 +1,23 @@
 # Copyright (c) 2024, NVIDIA CORPORATION.
 
 import os
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Union
```

Comment:
very very very minor thing, do whatever you want with it.

Since the only place this is used in this file is within the `if TYPE_CHECKING`, I think it'd be preferable to have the import of `Union` down in that block. To save a tiny tiny bit of import time for regular use of the tool.

Reply:
You're absolutely right, I should have done that from the get-go but forgot about that block. Done.